package.json

```diff
@@ -12,7 +12,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "test": "jest --passWithNoTests --silent",
+    "test": "snyk test && jest --passWithNoTests --silent",
```

Comment:
So this breaks the `test` command for anyone without access to our `snyk`, because you have to log in to run the test command after this. Probably it's fine for our team, but in the spirit of open-source, we should be trying to keep the codebase as working as possible without access to 3rd party services or secret adobe APIs, for example.

I would say, let's create a new command here instead, like `npm run snyk`.

Reply:
Good point. Fixed this on my branch.